### PR TITLE
Mhv 61572 add filter for rx vagov

### PR DESCRIPTION
**Pull request by robertbylight**
## Summary

made the attribute disp_status filterable

## Related issue(s)

[Filter backend work #4](https://jira.devops.va.gov/browse/MHV-61572)

## Testing done

spec test for get #index filter with pagination

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by vmenshutin-bylight — APPROVED** [review]

**Review by jperk51 — APPROVED** [review]
shouldn't cause any issues for mobile

File: spec/support/schemas/my_health/prescriptions/v1/prescription_list_filtered_with_pagination.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      "properties": {
+        "self": { "type": "string" },
+        "first": { "type": "string" },
+        "prev": { "type": ["string", "null"] },
+        "next": { "type": ["string", "null"] },
+        "last": { "type": "string" }
+      }
+    }
+  }
+}
```
Comment (RachalCassity) [inline spec/support/schemas/my_health/prescriptions/v1/prescription_list_filtered_with_pagination.json:50]: Please add newline at the end

**Review by RachalCassity — APPROVED** [review]

**Review by mmoyer-va — APPROVED** [review]
Looks good!

File: spec/support/schemas/my_health/prescriptions/v1/prescription_list_filtered_with_pagination.json (hunk 1 above)
Comment (robertbylight) [reply spec/support/schemas/my_health/prescriptions/v1/prescription_list_filtered_with_pagination.json:50]: hmm very odd i tried, however on my local the extra line is already there so there is nothing to push according to my git. I have some related work for vets-api  that ill be pushing soon, Ill see if I can add this new line on that pr after this is merged.

**Pull request merged**